lib/spack/spack/compiler.py

```diff
@@ -274,11 +268,12 @@ def __init__(self, cspec, operating_system, target,
         def check(exe):
             if exe is None:
                 return None
-            _verify_executables(exe)
             return exe
 
         self.cc  = check(paths[0])
         self.cxx = check(paths[1])
+        self.f77 = None
```

Comment:
The primary functional difference I see here is that now the `F77`/`FC` compilers will be checked: the old code looked suitable for checking `CC`/`CXX` but since I guess the Fortran compilers are not set here (weird but not an issue with this PR) your solution seems good for checking all of them.

Reply:
The old check didn't actually check anything because of a parenthesis error, but it did try to check F77/FC